assembl/models/action.py

```diff
@@ -293,8 +298,8 @@ def send_post_to_socket_ts(mapper, connection, target):
         & (_lpt.c.post_id == Content.__table__.c.id)
         & (_actt.c.type ==
            LikeSentimentOfPost.__mapper_args__['polymorphic_identity'])
-        & (_actt.c.tombstone_date == None)
-        ).correlate_except(_actt, _lpt), deferred=True)
+        & (_actt.c.tombstone_date is None)
```

Comment:
revert this change, in sqlalchemy `is None` is not supported

Reply:
it's weird ... I actually never did this change !!

Reply:
sorry my bad, I actually did "autopep8 --in-place --aggressive on that file ... this is the plugin I found to put everything on pep8